packages/cli/src/lib/commands/space.ts

```diff
@@ -30,6 +30,24 @@ export const space: CommandDefinition = (program) => {
                 throw new Error("Implement me");
             });
 
+    spaceCmd
+        .command("info")
+        /* TODO for future use
```

Comment:
I guess this we're leaving because it is for later implementation, the same as in `host info`?

Reply:
We leave for future implementation displaying info of a particular space (by giving its name|id ?? tbc). so you can check the info of any space, not only the one you are currently in. 
Yes, the same with `hub`.

Reply:
> I'm only thinking whether not to present the three json configs in an array. This way the result would be a valid json.

Good catch @alicja-gruzdz, thanks! I will put it in an object.